### DISCUSSION TITLE
Is this not possible to have a log file?

### DISCUSSION THREAD

**Discussion by lapinvert** [emmett-framework/granian#368] [Q&A] [answered]
Like `--access-log <path-to-file>` and `--error-log <path-to-file>` of gunicorn.

Is this not possible? and what should we use instead?

**Comment by gi0baro** [comment, marked as answer]
@lapinvert logging configuration in Granian follows the `logging` standard library configuration.
You just need to customise [the default handlers](https://github.com/emmett-framework/granian/blob/master/granian/log.py#L43) to use files in place of the stdout.

You can put such configuration in a JSON file and point Granian to it using the `--log-config` CLI option, or, in case you run `Granian` from Python, define a `dict` for that and pass it to the `Granian` class using the `log_dictconfig` parameter (https://github.com/emmett-framework/granian/blob/master/granian/server.py#L89).
